docs/firewall_opnsense.rst

```diff
@@ -0,0 +1,780 @@
+section, make sure that the **Block private networks** and **Block bogon networks**
+checkboxes are checked.
+
+Scroll down and click  **Save**, then click **Apply changes** when prompted.
+
+Configure the LAN interface
+'''''''''''''''''''''''''''''
+Next, navigate to **Interfaces > [LAN]**. In the **Basic configuration** section,
+check the checkbox labeled **Prevent interface removal**.
+
+In the **Generic configuration** section, select ``Static IPv4`` in the **IPv4
+Configuration Type** dropdown, and ``None`` in the **IPV6 Configuration Type**
+dropdown.
+
+Scroll down and click **Save**, then click **Apply changes** when prompted.
+
+Configure the OPT1 interface
+'''''''''''''''''''''''''''''
+Next, navigate to **Interfaces > [OPT1]**. In the **Basic configuration** section,
+check the checkbox labeled **Prevent interface removal**.
+
+In the **Generic configuration** section, select ``Static IPv4`` in the **IPv4
+Configuration Type** dropdown, and ``None`` in the **IPV6 Configuration Type**
+dropdown.
+
+Scroll down. In the **Static IPv4 Configuration** section, enter the *Application
+Gateway* IP address and routing prefix (``10.20.2.1`` and ``24`` if you are using
+the recommended values).
+
+Click **Save**, then click **Apply changes** when prompted.
+
+Configure the OPT2 interface
+'''''''''''''''''''''''''''''
+Finally, navigate to **Interfaces > [OPT2]**. In the **Basic configuration** section,
+check the checkbox labeled **Prevent interface removal**.
+
+In the **Generic configuration** section, select ``Static IPv4`` in the **IPv4
+Configuration Type** dropdown, and ``None`` in the **IPV6 Configuration Type**
+dropdown.
+
+Scroll down. In the **Static IPv4 Configuration** section, enter the *Monitor
+Gateway* IP address and routing prefix (``10.20.3.1`` and ``24`` if you are using
+the recommended values).
+
+Click **Save**, then click **Apply changes** when prompted.
+
+Configure Firewall Aliases
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to simplify firewall rule setup, the next step is to configure aliases
+for hosts and ports referred to in the rules.
+
+To start, first navigate to **Firewall > Aliases**. You should see some system-defined
+aliases as shown below:
+
+|OPNSense - Alias Start|
+
+Click the **+** button to add new aliases. You should add the aliases defined in
+the table below (assuming recommended values for IP addresses):
+
+.. list-table:: Firewall Aliases
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Content
+
+   * - admin_workstation
+     - Host(s)
+     - 10.20.1.2
+
+   * - app_server
+     - Host(s)
+     - 10.20.2.2
+
+   * - external_dns_servers
+     - Host(s)
+     - 8.8.8.8, 8.8.4.4
+
+   * - monitor_server
+     - Host(s)
+     - 10.20.3.2
+
+   * - local_servers
+     - Host(s)
+     - app_server, monitor_server
+
+   * - OSSEC
+     - Port(s)
+     - 1514
+
+   * - ossec_agent_auth
+     - Port(s)
+     - 1515
+
+   * - antilockout_ports
+     - Port(s)
+     - 80,443
```

Comment:
If it is syntactically valid, a space before the `,` would make this a bit easier to read (as you do with the DNS servers)

Reply:
fixed.

Reply:
I agree with you, @eloquence, in terms of readability, but see (e.g.) <https://github.com/freedomofpress/securedrop-docs/pull/272#discussion_r751824824> for my experience following this instruction as revised.  :-(

Reply:
double-fixed :)